### ISSUE TITLE
Warning this addon breaks AI with disableAI "MOVE" on init!

### ISSUE THREAD

**Issue by GodlikeRU** [sethduda/AdvancedUrbanRappelling#11]
Hello,

I encountered this behaviour in my mission and the other one from workshop.

If you place units with disableAI "MOVE" in init field and run the game on dedicated server with this mod enabled then they will end up like that - https://steamuserimages-a.akamaihd.net/ugc/169290952004214801/A1211376D73D47209FF221855650E18EBD0260F3/

Unable to move even if you enableAI  "MOVE"

I disabled this mod and Advanced Rappeling from heli too. 

I think You should look into this.

**Comment by akioku** [comment]
Yeah, I get this 100% of the time with DMS spawned AI on a Linux dedicated server running Exile. Had to disable the mod as you did.

**Comment by martinco** [comment]
I too experience the above on a mission I'm making (linux server) - and for all units spawned on the server the AI are stuck (they are still processing, shooting etc, just no animations/movement)

If you spawn on another host (without the mod) and transfer them to server, they remain fine - so my gut is to say related to animations (so probably a bis thing unless it doesn't like something in the pbo/pp without spitting anything to the rpt that i noticed)

Given that the anims + sounds within the mod are really for clients aesthetics/audio, I just took the server side init script and threw it on the server and called it from initServer - this giving my AI their movement back and allowing my players to rappel

**Comment by TYKUHN2** [comment]
Issue occurs on servers with no other mods on Linux v1.90 primarily visible during a Warlords mission

**Comment by Varrkan82** [comment]
The bug still present on Arma v.1.98 and Linux server

**Comment by Nerexis** [comment]
@martinco Is this related? https://github.com/A3Antistasi/A3-Antistasi/issues/196

Does it also happen with only Advanced Rappelling or only with Advanced Urban Rappeling?

**Comment by TechnoTroglodtye** [comment]
> I too experience the above on a mission I'm making (linux server) - and for all units spawned on the server the AI are stuck (they are still processing, shooting etc, just no animations/movement)
> 
> If you spawn on another host (without the mod) and transfer them to server, they remain fine - so my gut is to say related to animations (so probably a bis thing unless it doesn't like something in the pbo/pp without spitting anything to the rpt that i noticed)
> 
> Given that the anims + sounds within the mod are really for clients aesthetics/audio, I just took the server side init script and threw it on the server and called it from initServer - this giving my AI their movement back and allowing my players to rappel

Hey @martinco, 

I know this is old news, but how did you end up specifically doing this?
